### DISCUSSION TITLE
Requisitos parte 1

### DISCUSSION THREAD

**Discussion by martineyz** [IIC2133-PUC/2024-2#155] [Tarea 2] [answered]
¿Para la parte 1 es un requerimiento usar árboles o simplemente se recomiendan?

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, es una recomendación (y es la idea ya que es la materia en la que se enfoca que aprendan en esa tarea), ten ojo con el informe ya que las preguntas iban dirigidas a contestarlas con árboles, por lo que tendrás que dar un mayor detalle en caso de implementar otra solución.
